deepchem/feat/tests/test_roberta_tokenizer.py

```diff
@@ -16,8 +16,7 @@ def test_smiles_call():
                                 add_special_tokens=True,
                                 truncation=True)
     for emb in [embedding, embedding_long]:
-        assert 'input_ids' in emb.keys() and 'attention_mask' in emb.keys()
```

Comment:
Why are we removing asserts here?

Reply:
Because the `__call__` function in `RobertaFeaturizer` now returns the `input_ids` only, which are the actual embeddings of the molecules. The reason is that `_featurize_shard()` in`CSVLoader` is later called, in which the `features` List needs the embeddings only but not the attention masks (see [here](https://github.com/deepchem/deepchem/blob/eef32b047ac69a32093d8a51bca3a1283e2b2112/deepchem/data/data_loader.py#L430)). This is actually why I modified the `__call__` in `RobertaFeaturizer` so that it returns the input_ids only, unlike the `__call__` in the inherited HuggingFace's `RobertaTokenizerFast`, which returns the whole dict (input_ids and attention_mask). 

So the asserts are removed since `attention_mask` are not returned anymore, and the test asserts that the lists of `input_ids` for all the molecules all have the same length after specifying `padding=True`.